app/forms/waste_carriers_engine/contact_postcode_form.rb

```diff
@@ -4,9 +4,11 @@ module WasteCarriersEngine
   class ContactPostcodeForm < PostcodeForm
     attr_accessor :temp_contact_postcode
 
+    validates :temp_contact_postcode, "waste_carriers_engine/postcode": true
+
     def initialize(transient_registration)
       super
-      self.temp_contact_postcode = @transient_registration.temp_contact_postcode
+      self.temp_contact_postcode = transient_registration.temp_contact_postcode
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.temp_contact_postcode = transient_registration.temp_contact_postcode
```